### PR TITLE
made changes in transformer cronjob to push transformed data in a kafka topic

### PR DESCRIPTION
**Pull request by prerna-egov**

### CODE REVIEW

File: utilities/transformer-cronjob/transformer.py [hunk 1]
```diff
@@ -4,25 +4,44 @@
...
 
 gender_vs_beneficiary_map = {}
 
 # Current time in milliseconds
 current_time_millis = int(time.time() * 1000)
 
-# 30 days ago in milliseconds 
-thirty_days_ago_millis = current_time_millis - (30 * 24 * 60 * 60 * 1000)
+# 3 hour ago in milliseconds 
+thirty_days_ago_millis = current_time_millis - (3 * 60 * 60 * 1000)
```
Comment (shailesh-egov) [inline utilities/transformer-cronjob/transformer.py:30]: @prerna-egov why you added 3 days here?

Comment (prerna-egov) [reply utilities/transformer-cronjob/transformer.py:30]: @shailesh-egov It's not 3 days it's  3 hours

File: utilities/transformer-cronjob/transformerConfig.yaml [hunk 2]
```diff
@@ -34,4 +34,11 @@ spec:
                 value : "individual-index-v1"
               - name: ES_TRANSFORMER_PI_INDEX
                 value: "transform-ifms-pi-index"
+              - name: KAFKA_BROKER
+                valueFrom:
+                  configMapKeyRef:
+                    name: egov-config
+                    key: kafka-brokers
+              - name: TRANSFORMER_PI_KAFKA_TOPIC
```
Comment (shailesh-egov) [inline utilities/transformer-cronjob/transformerConfig.yaml:42]: @prerna-egov we can not hardcode the value, add under a condition.

File: utilities/transformer-cronjob/transformer.py (hunk 1 above)
Comment (shailesh-egov) [reply utilities/transformer-cronjob/transformer.py:30]: why you are using the same variable?

Comment (prerna-egov) [reply utilities/transformer-cronjob/transformer.py]: @shailesh-egov updated the variable name

File: utilities/transformer-cronjob/transformerConfig.yaml (hunk 2 above)
Comment (prerna-egov) [reply utilities/transformer-cronjob/transformerConfig.yaml:54]: @shailesh-egov added conditions

**Review by shailesh-egov — APPROVED** [review]

**Review by egov-arindam — APPROVED** [review]

**Pull request merged**